### PR TITLE
Cleanup InstanceGroup MixedInstancePolicy docs

### PR DESCRIPTION
**Pull request by gjtempleton**
Also improved consistency of rest of InstanceGroup docs

Cleans up MDLint errors throughout.

### CODE REVIEW

File: docs/instance_groups.md [hunk 1]
```diff
@@ -83,7 +84,7 @@ The procedure to resize the root volume works the same way:
 
 For example, to set up a 200GB gp2 root volume, your InstanceGroup spec might look like:
 
-```
+```YAML
 metadata:
   creationTimestamp: "2016-07-11T04:14:00Z"
```
Comment (hakman) [inline docs/instance_groups.md]: I think these creation timestamps can be removed also. I don't see much value to have them here.

Comment (gjtempleton) [reply docs/instance_groups.md]: Good point, only seems likely to confuse people. Addressed.

**Comment by rifelpet** [comment]
travis still isnt reporting its status properly. im going to try closing/reopening the PR to see if that fixes it but i'm not optimistic.

**Comment by rifelpet** [comment]
@gjtempleton still not working :( you may need to close this PR and open a new one entirely. Same with #8810

**Comment by gjtempleton** [comment]
Damn, very odd, will try that, thanks for the attempt.

**Pull request closed without merging**